### PR TITLE
[script.service.hue] 1.1.3

### PR DESCRIPTION
**Pull request by zim514**
### Description
v.1.1.3
- Move Qhue module to seperate package ([script.module.qhue](https://github.com/xbmc/repo-scripts/pull/2259))
- Localisation updates

v1.1.2
- Remove SSDP support (Deprecated by Philipps)
- Add toggle to disable Hue Labs when Ambilight is activated (Thanks @toupeira!)
- Refactoring (Pylint, PEP8, add copyright header and other style improvements)
- Bug and crash fixes
- Update Qhue lib
- Improve exception handling
- Add localisations/weblate support
- Refactor logging system
- Update to new settings system

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.service.hue/resources/lib/qhue/README.md [hunk 1]
```diff
@@ -6,15 +6,15 @@ I wrote it because some of the other (excellent) frameworks out there weren't qu
 
 ## Understanding Qhue
```
Comment (enen92) [inline script.service.hue/resources/lib/qhue/README.md:7]: why is this module bundled with the addon and not shared as a script.module? Is it not plain upstream code?

**Review by enen92 — COMMENTED** [review]
LGTM, question about the builtin module. Maybe consider removing it from the addon and submit it as a script.module addon.

File: script.service.hue/resources/lib/qhue/README.md (hunk 1 above)
Comment (zim514) [reply script.service.hue/resources/lib/qhue/README.md:7]: I didn't find any other addons that use this module so assumed it wouldn't be accepted, but if this is best practice I can change it

Comment (zim514) [reply script.service.hue/resources/lib/qhue/README.md]: Changed as requested, please see associated PR for [script.module.qhue](script.module.qhue).

**Review by enen92 — APPROVED** [review]

**Comment by zim514** [comment]
@enen92 Thank you for your time!

**Pull request merged**